src/SFML/Audio/SoundStream.cpp

```diff
@@ -234,11 +234,26 @@ SoundStream::~SoundStream() = default;
 
 
 ////////////////////////////////////////////////////////////
-SoundStream::SoundStream(SoundStream&&) noexcept = default;
+SoundStream::SoundStream(SoundStream&& rhs) noexcept : m_impl(std::move(rhs.m_impl))
```

Comment:
Any reason we don't implement move semantics for the impl?

Reply:
I'm not entirely sure what you mean here -- the impl is stored as a `std::unique_ptr`, so internally the type `Impl` is never moved nor copied.

Reply:
Ah, I missed that it was a unique_ptr